test/dummy/lib/design_system/components/layout/layout_facade.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class LayoutFacade < FenrirView::Presenter
+  property :title
+
+  # Layout section handling
+  attr_reader :layout
+
+  def layout=(context)
+    @layout = LayoutSectionCollector.new(context: context)
```

Comment:
The layout is actually equal to an instance of `LayoutSectionCollector`, not the context which is passed in to `layout=`, so maybe call this `build_layout_from_context(context)` or use `context=(context)` and `def layout; @layout ||= LayoutSectionCollector.new(context: context); end`